mteb/evaluation/evaluators/InstructionRetrievalEvaluator.py

```diff
@@ -0,0 +1,30 @@
+import logging
+from typing import Dict, List
+
+from .RetrievalEvaluator import (
+    RetrievalEvaluator,
+)
+
+logger = logging.getLogger(__name__)
+
+
+
+class InstructionRetrievalEvaluator(RetrievalEvaluator):
```

Comment:
This is now very short, inherits pretty much everything from the RetrievalEvaluator